setup.py

```diff
@@ -67,7 +67,7 @@ def find_files(pattern, path=None, root="rez"):
     entry_points={
         "console_scripts": get_specifications().values()
     },
-    include_package_data=True,
+    include_package_data=False,
     zip_safe=False,
```

Comment:
`setup` supports specifying a minimum python version -- https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#python-requires

```python
setup(
    python_requires=">=3.7",
    ...
)
```

Reply:
This would prevent users from pip install for using the API. Though we should specify python_requires=">=2.7".